{
	"id": "AVG-1477",
	"advisories": [
		{
			"content": {
				"id": "AVG-1477",
				"severity": [
					{
						"type": "vendor",
						"source": "security.archlinux.org",
						"vendor": "Critical"
					}
				],
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/AVG-1477"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-16044",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2020-16044"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21117",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21117"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21118",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21118"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21119",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21119"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21120",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21120"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21121",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21121"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21122",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21122"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21123",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21123"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21124",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21124"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21125",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21125"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21126",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21126"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21127",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21127"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21128",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21128"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21129",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21129"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21130",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21130"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21131",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21131"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21132",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21132"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21133",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21133"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21134",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21134"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21135",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21135"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21136",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21136"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21137",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21137"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21138",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21138"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21139",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21139"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21140",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21140"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-21141",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-21141"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "arch",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"vulnerable": true,
								"fix_status": {
									"class": "fixed"
								},
								"package": {
									"name": "chromium"
								},
								"affected": {
									"type": "pacman",
									"range": [
										{
											"lt": "88.0.4324.96-1"
										}
									],
									"fixed": [
										"88.0.4324.96-1"
									]
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "arch",
		"raws": [
			"vuls-data-raw-arch/AVG-1477.json"
		]
	}
}
